src/shared/components/Post/HighPriceRateBadge/HighPriceRateBadge.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from "classnames/bind";
+import styles from "@/shared/components/Post/HighPriceRateBadge/HighPriceRateBadge.module.scss";
+import ArrowUp from "@/images/ic_arrowUp.svg";
+
+const cn = classNames.bind(styles);
+
+interface HighPriceRateBadgeProps {
+  closed: boolean;
+  isPast: boolean;
+  hourlyPay: number;
+  originalHourlyPay: number;
+}
+
+export default function HighPriceRateBadge({ closed, isPast, hourlyPay, originalHourlyPay }: HighPriceRateBadgeProps) {
+  const percentage = Number((((hourlyPay - originalHourlyPay) / originalHourlyPay) * 100).toFixed());
+  let color;
+  if (percentage < 20) {
+    color = "red20";
+  } else if (percentage < 50) {
+    color = "red30";
+  } else {
+    color = "red40";
+  }
+  return (
```

Comment:
return문 전에는 라인을 하나 띄워주면 좋을 것 같습니다~
```suggestion
  }
  
  return (
```